index.js

```diff
@@ -6,6 +6,7 @@ var forEachEls = require("./lib/foreach-els.js")
 var newUid = require("./lib/uniqueid.js")
 
 var noop = require("./lib/util/noop")
+var contains = require("./lib/util/contains.js")
```

Comment:
I think these vars were originally alphabetically ordered, but grouping modules by path is also something that makes sense -- another thing to standardise on for #97?

Reply:
Good idea. I'll mention that there.